src/documents/management/commands/document_consumer.py

```diff
@@ -298,15 +298,16 @@ def handle_inotify(self, directory, recursive, is_testing: bool):
         else:
             descriptor = inotify.add_watch(directory, inotify_flags)
 
-        inotify_debounce: Final[float] = settings.CONSUMER_INOTIFY_DELAY
+        inotify_debounce_secs: Final[float] = settings.CONSUMER_INOTIFY_DELAY
```

Comment:
No reason to have create a  _secs variable imho, just create _ms one with settings.CONSUMER_INOTIFY_DELAY * 1000

Reply:
Oh scratch this

Reply:
Nope, we need seconds within the function body

```
waited_long_enough = (
                        monotonic() - last_event_time
                    ) > inotify_debounce_secs
```